extras/controllerClient/x86/example_autohotkey.ahk

```diff
@@ -0,0 +1,50 @@
+/**
+ * Because AutoHotkey's DllCall function can be a bit tricky for newcomers, I've taken the liberty of creating the following wrapper functions.
+ * But of course, you could just as easily call the DLL directly, as I have done inside the functions.  But it might be easier to save the wrapper
+ * functions to a separate file and then including them (AutoHotkey has a #include directive similar to C++).
```

Comment:
Oh, reading it again I see where the confusion came from. Where the comment says "you could just as easily call the DLL directly, as I have done inside the functions", I read that as talking about the example functions,rather than the wrapper functions.